cdc/processor/manager.go

```diff
@@ -189,6 +199,22 @@ func (m *Manager) handleCommand() error {
 	return nil
 }
 
+const pdTimeUpdateInterval = 1 * time.Second
+
+func (m *Manager) currentTimeFomPDCached(ctx context.Context) (int64, error) {
+	cdcCtx := ctx.(cdcContext.Context)
+	if time.Since(m.lastUpdatedPdTime) <= pdTimeUpdateInterval {
+		return m.pdPhysicalTimeCache, nil
+	}
+	physical, _, err := cdcCtx.GlobalVars().PDClient.GetTS(ctx)
+	if err != nil {
+		return physical, errors.Trace(err)
+	}
+	m.pdPhysicalTimeCache = physical
+	m.lastUpdatedPdTime = time.Now()
```

Comment:
It's similar to `gcManager.CurrentTimeFromPDCached()`. Could you extract a struct that manages TSO cache?

Reply:
done